spec/system/admin/order_spec.rb

```diff
@@ -644,10 +610,7 @@ def new_order_with_distribution(distributor, order_cycle)
           end
 
           it 'can send invoices' do
-            accept_alert "An invoice for this order will be sent to the customer. "\
-                         "Are you sure you want to continue?" do
-              click_link "Send Invoice"
-            end
```

Comment:
This is more than a style change. The whole spec seems rewritten. Was that intended? It would be better to explain such changes in a separate commit. But it could just be a merge conflict that wasn't resolved properly.